### DISCUSSION TITLE
Default Values for iOS and Android AR

### DISCUSSION THREAD

**Discussion by sercandemirtas** [google/model-viewer#1925] [Q&A]
Hello all,

What should be default values for both iOS and Android AR modes to run it properly. 

Ps: In this codes I shared below iOS AR works well but on Android device (Galaxy S9 - 10.0) something is wrong on plane detection. Object not appear on the floor. It appers on top and I cant move it to ground. 

My project link
https://confident-lamport-09f645.netlify.app/m2.html

 
                  class="a"
                  src="/Assets/m2/lamba.glb"
                  alt="3D Koltuk by BaseBros"
                  id="reveal" 
                  loading="eager"
                  poster="Assets/loading4.gif" 
                  background-color="#fff"
                  camera-controls
                  camera-orbit="-15deg 75deg m"
                  ar 
                  quick-look-browsers="safari chrome"
                  scene-viewer-browser="safari chrome"
                  autoplay
                  auto-rotate
                  ar-modes="scene-viewer quick-look"
                  ios-src="Assets/m2/lamba.usdz"
                  shadow-intensity="1" 
                  stage-light-intensity="1" environment-intensity="1"
                  >
                  <input slot="ar-button" class="ar-button" value="submit" type="image" 
                  src="Assets/ariconsiyah.png" width="30px" height="30px"> 
              

 

### Browser Affected

- [ X] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ x] Safari

### OS

- [x ] Android
- [ x] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

**Comment by elalish** [comment]
Try removing `ar-modes`, which will go back to the webXR default. If that works better, then it may be a scene-viewer bug. Also `scene-viewer-browser` is not an attribute.
